perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -77,14 +77,16 @@
 flags.DEFINE_boolean('against_device', False,
                      'Unmount the device\'s filesystem so we can test against '
                      'the raw block device. If --generate-scenarios is given, '
-                     'will generate a job file that uses the block device.')
+                     'will generate a job file that uses the block device. '
+                     'When using --against_device, default to pre-filling the '
+                     'device to 100% unless modified with --device_fill_size.')
 flags.DEFINE_string('device_fill_size', '100%',
```

Comment:
Tony, actually it's tricker than it appears... What we want is:

1) if the user is testing against the device directly, the default value of --device_fill_size should be 100% because in this mode, the user should fill the device entirely before the fio run.

2) If the user is testing using a file system, then the default behavior should be no-fill (0%) because that is unnecessary as fio will create the test file first and use that file for testing. However, on some providers like AWS, they require pre-warming of the device no matter what, so we still need to provide the user a choice to specify --device_fill_size to be 100% when they run fio on those providers in file system mode.

To sum it up: the desired default value of this flag changes as the scenario changes. 

After some discussion, we figured out a smart way to handle this: we will simply check for the presence of the flag in the test-with-file-system mode case, if the flag is present, then we will follow user's order, otherwise, we won't fill the device. In the test-against-device mode, we will just use the flag as is with the default value being 100%. (so if the flag is not present in the test-against-device mode, the default value of 100% will apply).